libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -166,25 +185,33 @@ - (id)initWithConfig:(SFHybridViewConfig *)viewConfig
 
 - (UIView *)newCordovaViewWithFrame:(CGRect)bounds
 {
-
-    // Setting our WKWebView based plugin as the engine to be used.
-    return [self newCordovaViewWithFrameAndEngine:bounds webViewEngine:@"SFWKWebViewEngine"];
+    if (self.useWKWebView) {
```

Comment:
Sets the correct `WebViewEnginePlugin` to use, based on the choice of `WebView`.